front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx

```diff
@@ -51,7 +52,7 @@ const SensorDeviceType = ({ children, ...props }) => (
       </td>
     )}
     {props.deviceFeature.category === DEVICE_FEATURE_CATEGORIES.MOTION_SENSOR && (
-      <td class="text-right">
+      <td class={cx('text-right', { 'text-nowrap': props.deviceFeature.last_value })}>
```

Comment:
same here

Reply:
Here it should be `last_value_changed` not `last_value`.